packages/apputils/src/vdom.ts

```diff
@@ -108,7 +108,7 @@ abstract class VDomRenderer<T extends VDomRenderer.IModel | null> extends Widget
 
 
 export
-class ElementVDomRenderer extends VDomRenderer<any> {
+class ReactElementWidget extends VDomRenderer<any> {
```

Comment:
Mind adding comments for the class and the constructor, and moving the private declaration out of the constructor for conformity?

Reply:
Done. Do the comments look OK?